src/less/breadcrumb/ds6/breadcrumb-base.less

```diff
@@ -0,0 +1,34 @@
+.breadcrumb {
+	list-style: none;
```

Comment:
Please ensure that all properties in LESS source files are ordered alphabetically.